docker-compose.yml

```diff
@@ -12,12 +12,18 @@ services:
     image: rediscommander/redis-commander:latest
     environment:
       - REDIS_HOSTS=local:redis:6379
-      - HTTP_USER=${USER_NAME:?error}
-      - HTTP_PASSWORD=${PASSWD:?error}
+      - HTTP_USER=root
+      - HTTP_PASSWORD=1234
```

Comment:
This is not supposed to be hard-coded. Change it to what's currently on main.

Reply:
Just pushed a commit to change that, ty for pointing it out